### ISSUE TITLE
create sample fixtures of NDNP data

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#61]
**Descriptive Summary**
In order to write tests for our NDNP batch ingest code, we will need fixtures to run the tests against. We should create a small mini batch of data in `./spec/fixtures/files/ndnp` that includes enough data to successfully test the creation of titles, issues, pages, and attached derivatives.

**Background**
Batch ingest of NDNP-spec-compliant digitized materials is an important use case voiced by a number of stakeholders that we need to support.

**This ticket will be complete when**
There is a batch of data in  `./spec/fixtures/files/ndnp` that includes files and directory structures matching [LoC's data/batches](https://chroniclingamerica.loc.gov/data/batches/) format to successfully test the creation of titles, issues, pages, and attached derivatives.

**Comment by ebenenglish** [comment]
We have decided to accomplish this by creating a separate gem to manage the fixtures. This decision was based on the fact that the fixtures may end up comprising >200MB, which isn't something that most people are going to want to add to their app.

By adding a separate fixtures gem as a `:test` and/or `:dev` dependency only, it will allow us to access the fixtures without forcing adopters to add all the files to their local installation.

[newspaper_works_fixtures](https://github.com/marriott-library/newspaper_works_fixtures) gem has been created.

**This ticket will be complete when** (updated)
`NewspaperWorksFixtures` gem has been added to `NewspaperWorks` as a development dependency in `newspaper_works.gemspec`...
```ruby
spec.add_development_dependency 'newspaper_works_fixtures', git: 'https://github.com/marriott-library/newspaper_works_fixtures'
```
... and we have verified that the fixture paths are accessible, and the changes have passed in CI.
